bin/terminus.php

```diff
@@ -38,6 +41,13 @@
     ->withArgument('fileCache');
 $container->inflector(SessionAwareInterface::class)
     ->invokeMethod('setSession', ['session']);
+$container->share('sites', Sites::class);
+$container->inflector(SiteAwareInterface::class)
+    ->invokeMethod('setSites', ['sites']);
+$container->share('user', User::class);
```

Comment:
I don't think it makes sense to put the current active user into the container like this.  I think it's better to ask the session for it.  As @TeslaDethray said above, a User could contain a reference to some other user, we don't know who the active user is when we are building the container, and if we want to lazy-evaluate who the user is, then the User class would need a reference to the session class, which is backwards.

It would be possible to make a `CurrentUser` class that has a reference to the session class and gets the current user, but it seems unnecessary to factor that responsibility out of the session class.